conf/machine/raspberrypi0-2w.conf

```diff
@@ -0,0 +1,17 @@
+#@TYPE: Machine
+#@NAME: RaspberryPi0 2 Wifi Development Board (32bit)
+#@DESCRIPTION: Machine configuration for the RaspberryPi0 2 Wifi in 32 bits mode
+
+MACHINEOVERRIDES = "raspberrypi3:${MACHINE}"
```

Comment:
Shall we define this after the include? Just to be safe.

Reply:
done